modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/Slice2DIndex.java

```diff
@@ -159,14 +175,17 @@ public synchronized Slice2DIndex getSlice2DIndex( int imageIndex ) throws IOExce
             long endDataPosition = raf.readLong();
 
             raf.seek(dataPosition);
-            int tIndex = raf.readInt();
-            int zIndex = raf.readInt();
+            int dimensions = raf.readInt();
```

Comment:
Sorry, I didn't notice that before:
will this change break already configured layer in GeoServer?
I'm wondering if the new code running against a .idx being configured by previous code will throws exceptions or refers wrong values.
OFF TOPIC: Andrea recently changed the sampleImage being stored by ImageMosaic and things stopped working on already configured layers, so he needed to introduce backward-compatibility fix and a different naming for the new file. 
Wondering if this slices change needs similar checks.